Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -11,6 +11,11 @@ namespace NoOpRunner.Core
 {
     public class NoOpRunner
     {
+        //That's silly, need static class or singleton for such parameters, using resources would be overkill 
```

Comment:
Yep, that sounds good. Just create a static game parameter class and put the constants there